lib/pf/Switch/Aruba.pm

```diff
@@ -482,6 +533,7 @@ sub radiusDisconnect {
         my $role = $roleResolver->getRoleForNode($mac, $self);
 
         my $acctsessionid = node_accounting_current_sessionid($mac);
+        my $node_info = node_attributes($mac);
```

Comment:
This can be removed and you can remove use pf::node qw(node_attributes); in the begining of the file.